<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1521</width>
     <height>1161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="PlotWidget" name="graph" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>900</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>175</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>60</y>
           <width>89</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>90</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>120</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>70</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>70</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>970</x>
           <y>20</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QTreeView" name="treeView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>300</width>
           <height>500</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>500</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="plot_btn">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>570</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
        <widget class="QPushButton" name="color_grids_btn">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>620</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Colorgrids</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>800</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> Enable Grid Selection</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>840</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> Enable Test Selection</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>540</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>540</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>540</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="refresh_grid_color_btn">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>660</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>RefreshGridColor</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="grid_btn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>600</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Grids</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="test_btn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>640</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Tests</string>
         </property>
        </widget>
        <widget class="QComboBox" name="combo_list_btn">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>770</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="repaint_tests_btn">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>800</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>repaint</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>740</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Select Options</string>
         </property>
        </widget>
        <widget class="QPushButton" name="grid_passing_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>880</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Shpw Passing</string>
         </property>
        </widget>
        <widget class="QPushButton" name="compare_btn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>900</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>compare</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphqt</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
